.github/workflows/mysql-migtests.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         yes | ./installer_scripts/install-yb-voyager --install-from-local-source
       env:
         ON_INSTALLER_ERROR_OUTPUT_LOG: Y
-        DEBEZIUM_VERSION: latest
+        DEBEZIUM_VERSION: priyanshi_csv-format
```

Comment:
You must fix these before you merge the PR.

Reply:
yes will do.